### DISCUSSION TITLE
How to upgrade Bootstrap

### DISCUSSION THREAD

**Discussion by NathanDotTo** [structurizr/ui#27] [Q&A]
Hi, it looks like the version of Bootstrap used in the UI project is out of date, as revealed by DAST scans.

The latest version of Bootstrap appears to be 5.1.3. It *looks* like we have 3.3.7 in the UI project.

I am not an expert on Bootstrap at all, so some guidance on how to upgrade would be appreciated please.

I have read this discussion that suggests that an upgrade is not really the right concept to think about:

https://stackoverflow.com/questions/68004792/how-to-fix-layout-changes-from-bootstrap-upgrade-from-3-to-5

Many thanks

Nathan

**Comment by simonbrowndotje** [comment]
Yes, Bootstrap 3 to 5 is essentially a "rewrite" of the front-end. This repo only contains the UI parts that are shared between the cloud service, on-premises installation, and Lite ... so the front-end of those apps would need to be modified too.
